iscript/src/iscript/macvpn.py

```diff
@@ -64,20 +65,133 @@ async def _sign_app(config, sign_config, app, entitlements_url, provisionprofile
     """
     # We mock the task for downloading entitlements and provisioning profiles
     entitlements_path = await download_entitlements_file(config, sign_config, {"payload": {"entitlements-url": entitlements_url}})
+
+    # TODO: Add provisionprofile_dir to scriptworker config
+    # https://github.com/mozilla-releng/scriptworker/blob/master/src/scriptworker/constants.py#L34
```

Comment:
Looks like Aki might be pushing back on this. Just remember to remove this `TODO` if it doesn't end up landing there.